R/cost_function.R

```diff
@@ -38,8 +36,11 @@ cost_function <- function(params, time, m, n, Nm, Nd,
                         frac_P = frac_P,
                         frac_k = frac_k)
 
+  pool_weight = sd(m) / m_prec # Normalization factor for pool size, see Eq. 12
+  ap_weight = sd((n / n + m) * 100) / ap_prec # Normalization factor for isotopic signature, see Eq. 13
```

Comment:
@kendalynnm Hmm this looks suspicious; `n / n + m` is just `1 + m` ...are the parentheses misplaced?

Reply:
Good catch, yes

Reply:
☝️ Once this is fixed, I would say go ahead and merge!